test/Microsoft.AspNetCore.Routing.Tests/Template/TemplateBinderTests.cs

```diff
@@ -594,7 +594,7 @@ public void GetUrlVerifyEncoding()
                 new RouteValueDictionary(new { controller = "Home" }),
                 new RouteValueDictionary(new { controller = "home", action = "Index", id = (string)null }),
                 values,
-                "/%23;%3F%3A@%26%3D%2B$,.mvc/showcategory/123?so%3Frt=de%3Fsc&maxPrice=100");
+                "/%23;%3F:@&=+$,%2F.mvc/showcategory/123?so%3Frt=de%3Fsc&maxPrice=100");
```

Comment:
@Tratcher - please confirm that this is right 👍 - I think PathString is less aggressive than UrlEncoder on purpose.

Reply:
👍